exporter/opentelemetry-exporter-otlp-proto-common/tests/test_log_encoder.py

```diff
@@ -287,6 +305,36 @@ def get_test_logs(
                         ),
                     ],
                 ),
+                PB2ResourceLogs(
+                    resource=PB2Resource(),
+                    scope_logs=[
+                        PB2ScopeLogs(
+                            scope=PB2InstrumentationScope(
+                                name="last_name",
+                                version="last_version",
+                            ),
+                            log_records=[
+                                PB2LogRecord(
+                                    time_unix_nano=1644650584292683009,
+                                    observed_time_unix_nano=1644650584292683010,
+                                    trace_id=_encode_trace_id(
+                                        212592107417388365804938480559624925555
+                                    ),
+                                    span_id=_encode_span_id(
+                                        6077757853989569445,
+                                    ),
+                                    flags=int(TraceFlags(0x01)),
+                                    severity_text="INFO",
+                                    severity_number=SeverityNumber.INFO.value,
+                                    body=_encode_value(
+                                        {"error": None}, allow_null=True
```

Comment:
would it result in having `error` in the body with null value or it won't be added?

Reply:
It will be encoded like this:

```
      body {
        kvlist_value {
          values {
            key: "error"
          }
        }
      }
```